content/includes/header.html

```diff
@@ -131,7 +131,11 @@
   <!-- /header alert box -->
   <div class="row top-banner">
     <div class="">
-      <h1 role="banner" class="va-header-logo"><a href="/">Vets.gov</a></h1>
+      <div role="banner" class="va-header-logo">
+        <a href="/">
+          <svg class="vets-logo" width="206" height="50" viewBox="0 0 206 50" xmlns="http://www.w3.org/2000/svg" pointer-events="none"><title>Vets dot gov from the U S Department of Veteran Affairs</title><g fill="none" fill-rule="evenodd"><g class="vets-logo-vetsdotgov"><path d="M194.04 37h3.84l6.88-19.44h-3.24l-3.68 11.04c-.56 1.92-1.2 3.88-1.76 5.72h-.16c-.6-1.84-1.24-3.8-1.8-5.72l-3.68-11.04h-3.4l7 19.44zM167.32 27.32c0 6.44 4.24 10.16 9 10.16s9-3.72 9-10.16c0-6.52-4.24-10.24-9-10.24s-9 3.72-9 10.24zm3.4 0c0-4.48 2.28-7.52 5.6-7.52 3.36 0 5.6 3.04 5.6 7.52 0 4.44-2.24 7.44-5.6 7.44-3.32 0-5.6-3-5.6-7.44zM148.28 40.72c0 3.36 3.28 5.24 8.04 5.24 6.04 0 9.84-3.12 9.84-6.72 0-3.24-2.28-4.64-6.8-4.64h-3.76c-2.64 0-3.44-.88-3.44-2.12 0-1.08.56-1.72 1.24-2.32.88.44 1.96.68 2.92.68 3.96 0 7.12-2.6 7.12-6.76 0-1.64-.68-3.12-1.6-4h4v-2.52h-6.76c-.72-.28-1.68-.48-2.76-.48-3.96 0-7.32 2.68-7.32 6.92 0 2.32 1.24 4.2 2.52 5.24v.16c-1 .68-2.12 1.96-2.12 3.6 0 1.52.76 2.56 1.72 3.16v.16c-1.76 1.2-2.84 2.76-2.84 4.4zm8.04-12.08c-2.24 0-4.12-1.8-4.12-4.64s1.84-4.52 4.12-4.52c2.28 0 4.12 1.68 4.12 4.52s-1.88 4.64-4.12 4.64zm-5.16 11.64c0-1.12.6-2.28 2-3.28.84.24 1.76.32 2.52.32h3.36c2.52 0 3.92.6 3.92 2.4 0 2.04-2.44 3.96-6.16 3.96-3.52 0-5.64-1.32-5.64-3.4zM137.52 33.8c0 2.08 1.52 3.68 3.56 3.68s3.56-1.6 3.56-3.68c0-2.12-1.52-3.72-3.56-3.72s-3.56 1.6-3.56 3.72zM118.8 34.72c2 1.64 5 2.76 7.6 2.76 5.32 0 8.16-2.8 8.16-6.4 0-3.64-2.84-5.04-5.36-5.96-2.04-.76-3.88-1.24-3.88-2.52 0-1 .72-1.56 2.24-1.56 1.4 0 2.76.64 4.2 1.68l2.64-3.52c-1.72-1.28-3.96-2.52-7-2.52-4.6 0-7.6 2.52-7.6 6.2 0 3.28 2.84 4.92 5.24 5.84 2.04.8 4.04 1.4 4.04 2.72 0 1.04-.76 1.68-2.52 1.68-1.68 0-3.32-.72-5.12-2.08l-2.64 3.68zM106.64 29.8c0 4.6 1.92 7.68 6.84 7.68 1.92 0 3.36-.4 4.4-.72l-.92-4.28c-.52.2-1.24.4-1.88.4-1.56 0-2.56-.92-2.56-3.16v-7.96h4.76v-4.6h-4.76v-5.28h-4.88l-.68 5.28-3.04.24v4.36h2.72v8.04zM85.28 27.08c0 6.56 4.36 10.4 10.04 10.4 2.36 0 5-.84 7.04-2.24l-1.96-3.56c-1.44.88-2.8 1.32-4.28 1.32-2.64 0-4.64-1.32-5.16-4.32h11.88c.12-.48.24-1.44.24-2.48 0-5.4-2.8-9.52-8.44-9.52-4.76 0-9.36 3.96-9.36 10.4zm5.64-2.04c.4-2.64 2-3.88 3.84-3.88 2.36 0 3.32 1.6 3.32 3.88h-7.16zM70.56 37h7.08l7.88-26.08h-6.04L76.4 23.08c-.76 2.84-1.28 5.48-2.08 8.36h-.16c-.8-2.88-1.32-5.52-2.08-8.36l-3.16-12.16h-6.24L70.56 37zM53 5h1v40h-1z"/></g><g class="vets-logo-va"><path d="M38.12 37h6.24l-8.2-26.08h-7.04L20.92 37h6l1.6-6.2h8l1.6 6.2zm-8.4-10.8l.64-2.4c.72-2.6 1.44-5.64 2.04-8.4h.16c.72 2.72 1.4 5.8 2.12 8.4l.64 2.4h-5.6zM9.56 37h7.08l7.88-26.08h-6.04L15.4 23.08c-.76 2.84-1.28 5.48-2.08 8.36h-.16c-.8-2.88-1.32-5.52-2.08-8.36L7.92 10.92H1.68L9.56 37z"/></g></g></svg>
```

Comment:
Instead of having the SVG code mixed into HTML, can you use an include to pull it in?